docs/content/en/docs/getting-started/baremetal/bare-spec.md

```diff
@@ -172,7 +172,7 @@ Optional field to skip deploying the control plane load balancer. Make sure your
 Refers to the Kubernetes object with Tinkerbell-specific configuration. See `TinkerbellDatacenterConfig Fields` below.
 
 ### kubernetesVersion (required)
-The Kubernetes version you want to use for your cluster. Supported values: `1.31`, `1.30`, `1.29`, `1.28`, `1.27`
```

Comment:
Can we link the announcement instead so we aren't repeating these words? Something like "Known bug for versions with `0` at the end" or something?

Reply:
Done